.github/workflows/tests-polkadot-deps.yml

```diff
@@ -61,7 +61,9 @@ jobs:
         node-version: [18.x, 20.x]
         required: ['required']
         include:
-          - node-version: [16.x, 22.x]
+          - node-version: 16.x
+            required: 'optional'
```

Comment:
I think Node 16 was not optional, so it should not be optional now either. It's ok to be optional for Node 22 tho.

Reply:
It was optional before: https://github.com/KILTprotocol/sdk-js/blob/2ea03c967efeb9ef737413d4a0407bb8d185cce9/.github/workflows/tests.yml#L86